force-app/main/default/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -682,11 +682,11 @@ public without sharing class UTIL_CustomSettingsFacade {
     }
 
     private static void configHouseholdNamingSettings(Household_Naming_Settings__c hns) {
-        hns.Household_Name_Format__c = '{!LastName} ' + label.npo02.DefaultHouseholdName;
+        hns.Household_Name_Format__c = '{!LastName} ' + 'Household';
```

Comment:
Interesting, so the Setting needs to be in English, but I assume the actual Naming logic uses translations?

Reply:
After discussing with @daniel-fuller, we're going to revert this change and instead update the Unit test to work with non-English orgs.